### PR TITLE
[NUI][VideoView] Fix VideoView guide

### PR DESCRIPTION
**Pull request by pkalota**
### Change Description ###

- Provide one link at the beginning to `VideoView` latest API
- Add information about supported formats
- Remove note about unnecessary plugin
- Change example image of `VideoView`
- Fix a little `Create VideoView` section
- Add `VideoView methods` section
- Add `VideoView Event` section

### CODE REVIEW

File: docs/application/dotnet/guides/nui/videoview.md [hunk 1]
```diff
@@ -1,58 +1,99 @@
...
+---
+
 # VideoView
 
-The `VideoView` class is a control for video playback and display. It provides minimum functions for playback (play, pause, stop, forward, and backward). Some options, such as volume, can be controlled through the control properties.
+The [VideoView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.VideoView.html) class is a control for video playback and display. It provides minimum functions for playback (play, pause, stop, forward, and backward). Some options, such as volume, can be controlled through the control properties. NUI supports many kinds of video format, such as `.avi`, `.3gp` and `mp4`.
 
  > **Note**  
- > You must install a video plugin to enable `VideoView`.
+ > The VideoView control does not use any privileges APIs on its own. However, if you use video files in a specific device storage, the application requires privileges to access the storage.
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:10]: `VideoView `

File: docs/application/dotnet/guides/nui/videoview.md [hunk 2]
```diff
@@ -1,58 +1,99 @@
...
-## VideoView event
+1.  To use the `VideoView` class, add the following namespaces:
 
-The following table lists the basic signal provided by the `VideoView` class.
+    ```cs
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    ```	    
+  
+2.   The video file is assumed to be in the resources directory. Create an instance of the `VideoView` class and use the `ResourceUrl` property to pass the path to the video file. Set `WidthResizePolicy` and `HeightResizePolicy` to make video use full window width and adjust height to width. In the end invoke `Play()` method to start video:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:28]: Set `WidthResizePolicy` and `HeightResizePolicy` to make video use the full window width and adjust height to width. 

adjust height to width? This part is not clear could you rephrase the above sentence

File: docs/application/dotnet/guides/nui/videoview.md [hunk 3]
```diff
@@ -1,58 +1,99 @@
...
 
-| Input signal  | Description                                 |
-| ------------- | ------------------------------------------- |
-| `Finished`    | Emitted when a video playback is finished.  |
+      Window.Instance.Add(player);
+      ```
 
-The `VideoView` class provides the `Finished`, which is emitted when the video playback is finished. The related callback can support some basic actions.
+> **Note**  
+> You can set the video file to be played in the `new VideoView()` function, or by modifying `Video` property.
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:41]: `Video` property.

why is Video in code tag?

Comment (pkalota) [reply docs/application/dotnet/guides/nui/videoview.md:41]: Because it is property `player.Video = ...`, similiar to the previous `player.ResourceUrl = ...`

File: docs/application/dotnet/guides/nui/videoview.md [hunk 4]
```diff
@@ -1,58 +1,99 @@
...
 
-<a name="2"></a>
-## Creating a VideoView
 
-The following basic example shows how to create a `VideoView` object:
+## VideoView Methods
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:44]: ## VideoView methods

We follow sentence case for headings, hence m in lower case

File: docs/application/dotnet/guides/nui/videoview.md [hunk 5]
```diff
@@ -1,58 +1,99 @@
...
-VideoView view = new VideoView( "videofile.mp4" );
-window.Add(view);
-view.ParentOrigin = ParentOrigin.Center;
-view.PivotPoint = PivotPoint.Center;
-view.HeightResizePolicy = ResizePolicyType.UseNaturalSize;
-view.WidthResizePolicy = ResizePolicyType.UseNaturalSize;
-view.Size2D = new Size2D( WIDTH, HEIGHT );
-view.Play();
-```
+  As it was mentioned before, the `VideoView` provides minimum functions for playback:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:46]: Can we rephrase it like this:
Are all the functions listed or only few of them listed ?
The following are the `VideoView `functions for playback:

File: docs/application/dotnet/guides/nui/videoview.md (hunk 3 above)
Comment (annie-samsung) [reply docs/application/dotnet/guides/nui/videoview.md:41]: @pkalota thanks for claryfing minor change please add the:

 You can set the video file to be played in the `new VideoView()` function, or by modifying the `Video` property.

File: docs/application/dotnet/guides/nui/videoview.md [hunk 6]
```diff
@@ -1,58 +1,99 @@
...
 
- > **Note**  
- > You can set the video file to be played in the `new VideoView()` function, or by modifying `Video` property.
+  1. The `Play()` method starts video:
+
+      ```cs
+      player.Play();
+      ```
+
+  2. The `Pause()` method pauses video. Video can be resumed by using the `Play()` method:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:54]: 2. The `Pause()` method pauses video.  To resume the video, use the `Play()` method:

File: docs/application/dotnet/guides/nui/videoview.md [hunk 7]
```diff
@@ -1,58 +1,99 @@
...
+
+      ```cs
+      player.Pause();
+      ```
+
+  2. The `Stop()` method stops video. After the `Play()` method is called, the video will be started from the beginning:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:60]: 3. The `Stop()` method stops video. To play the video, use the `Play()` method, the video will play from the  beginning:

File: docs/application/dotnet/guides/nui/videoview.md [hunk 8]
```diff
@@ -1,58 +1,99 @@
...
+
+      ```cs
+      player.Stop();
+      ```
+
+  2. The `Forward(x)` method rewind video forward for `x` miliseconds:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:66]: 4. The `Forward(x)` method fast forwards the video for `x` milliseconds:

File: docs/application/dotnet/guides/nui/videoview.md [hunk 9]
```diff
@@ -1,58 +1,99 @@
...
+
+      ```cs
+      player.Forward(1000); // +1 second
+      ```
+
+  2. The `Backward(x)` method rewind video backward for `x` miliseconds:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:72]: 5. The `Backward(x)` method rewinds the video backward for `x` milliseconds:

File: docs/application/dotnet/guides/nui/videoview.md [hunk 10]
```diff
@@ -1,58 +1,99 @@
...
+
+      ```cs
+      player.Backward(1000); // +1 second
+      ``` 
+
+## VideoView Event
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:78]: ## VideoView event

File: docs/application/dotnet/guides/nui/videoview.md [hunk 11]
```diff
@@ -1,58 +1,99 @@
...
+      ```
+  
+  2. Add handler to `VideoView` instance:
+
+      ```cs
+      player.Finished += OnFinish;
+      ```
 
-<a name="3"></a>
 ## VideoView Properties
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:97]: ## VideoView properties

File: docs/application/dotnet/guides/nui/videoview.md [hunk 12]
```diff
@@ -63,15 +104,13 @@ The following table lists the available `VideoView` properties.
...
+| `Video`       | PropertyMap  | Video file URL string. This property can also provide additional features, such as a custom shader, by `PropertyMap`. |
 | `Looping`     | Boolean      | Enable or disable the looping of the playback. |
 | `Muted`       | Boolean      | Whether the playback is muted.            |
 | `Volume`      | PropertyMap  | Playback volume. The `PropertyMap` must get left and right volume scalar as a float type. |
 | `Underlay`    | Boolean      | Set the underlay either as `true` or `false`, to allow NUI(DALi) to draw videos on either **Underlay** or **Overlay**. |
 | `ResourceUrl` | String       | The video file URL as a string type.            |
 
-
-
 ## Related Information
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:114]: ## Related information

File: docs/application/dotnet/guides/nui/videoview.md [hunk 13]
```diff
@@ -1,58 +1,99 @@
...
-## VideoView event
+1.  To use the `VideoView` class, add the following namespaces:
 
-The following table lists the basic signal provided by the `VideoView` class.
+    ```cs
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    ```	    
+  
+2.   The video file is assumed to be in the resources directory. Create an instance of the `VideoView` class and use the `ResourceUrl` property to pass the path to the video file. Set `WidthResizePolicy` to make `VideoView` instance use full width of the window and set `HeightResizePolicy` to maintain aspect ratio of video. In the end invoke `Play()` method to start video:
```
Comment (annie-samsung) [inline docs/application/dotnet/guides/nui/videoview.md:28]: In the end invoke `Play()` method to start video:

It could be a bit confusing to see end & start in the same line, hence recommend to change to:

And then you can invoke the `Play()` method to start video:

**Review by annie-samsung — COMMENTED** [review]
Please find my comments.

File: docs/application/dotnet/guides/nui/videoview.md (hunk 5 above)
Comment (pkalota) [reply docs/application/dotnet/guides/nui/videoview.md:46]: Yes, all VideoView specific functions are listed here.

**Review by annie-samsung — APPROVED** [review]
looks good hence approving

**Review by mijin85cho — COMMENTED** [review]
Hello, @annie-samsung 
Would you please check these points also? This PR should not be approved & merged as there is a broken image.

File: docs/application/dotnet/guides/nui/videoview.md [hunk 14]
```diff
@@ -1,58 +1,99 @@
...
+  2. The `Forward(x)` method rewind video forward for `x` miliseconds:
+
+      ```cs
+      player.Forward(1000); // +1 second
+      ```
+
+  2. The `Backward(x)` method rewind video backward for `x` miliseconds:
+
+      ```cs
+      player.Backward(1000); // +1 second
```
[the remark this answers is not in the document]
Comment (pkalota) [reply docs/application/dotnet/guides/nui/videoview.md:75]: Yes, you are right, will be changed.

**Comment by annie-samsung** [comment]
> Hello, @annie-samsung
> Would you please check these points also? This PR should not be approved & merged as there is a broken image.

@mijin85cho, thanks for pointing that out, will wait for fixes.

File: docs/application/dotnet/guides/nui/videoview.md [hunk 15]
```diff
@@ -63,15 +104,13 @@ The following table lists the available `VideoView` properties.
 
 | Property      | Type         | Description                              |
 | ------------- | ------------ | ---------------------------------------- |
-| `Video`       | PropertyMap  | Video file URL string. This property can also provide additional features, such as a custom shader, by `Property::Map`. |
+| `Video`       | PropertyMap  | Video file URL string. This property can also provide additional features, such as a custom shader, by `PropertyMap`. |
 | `Looping`     | Boolean      | Enable or disable the looping of the playback. |
 | `Muted`       | Boolean      | Whether the playback is muted.            |
 | `Volume`      | PropertyMap  | Playback volume. The `PropertyMap` must get left and right volume scalar as a float type. |
 | `Underlay`    | Boolean      | Set the underlay either as `true` or `false`, to allow NUI(DALi) to draw videos on either **Underlay** or **Overlay**. |
```
[the remark this answers is not in the document]
Comment (pkalota) [reply docs/application/dotnet/guides/nui/videoview.md:111]: Yes, in this case it will be fixed

**Review by annie-samsung — APPROVED** [review]
All comments are fixed hence approving
